app/client/lib/highlight.js

```diff
@@ -1,5 +1,4 @@
 var find = require('findandreplacedomtext')
-var css = require('dom-css')
 
 // Highlights all the words in text inside a node
 function Highlight (node, text, wrapper) {
```

Comment:
@blahah thank you so much for the suggestions, I'm still figuring out everything on the code base but I hope I get more acquainted with everything soon 😊 I'm doing all the suggested changes and will PR soon. 

Regarding the `text.trim() === '*'` comparison, I was thinking about doing it after validating that it's not an array and doing it implicitly by validating that it's not an empty string after replacing the `*` and trimming, like this:

```
if (!Array.isArray(text)) {
  text = text.replace('*', '').trim()
  if (!text) return
  text = text.split(' ')
}
```

What do you think?